geojsonvt.gyp

```diff
@@ -42,6 +42,7 @@
           '<@(variant_cflags)',
           '<@(geometry_cflags)',
           '<@(rapidjson_cflags)',
+          '${CXX_FLAGS}',
```

Comment:
Typically, the environment variable is called `CXXFLAGS` (without the underscore). Any reason this one has the underscore?